scripts/script.ts

```diff
@@ -182,28 +183,43 @@ function createPlayPage() {
 function createEndPage() {
   gamePage = GamePage.EndPage;
   const mainWrapper = clearMainWrapper();
+  connectUsernameWithGuesses();
+  let totalGuesses = localStorage.getItem("score");
 
   const markup = `
     <div class="title_ender">
     <h2>YOU WON!</h2>
     </div>
     
     <div class="high_score">
-    <h2>HIGHEST SCORES</h2>
-    <img src="./assets/images/win.gif" alt="" class="images_game" />
-    <div class="gameEndMessage"> "Only ${nGuesses} ${gameText.correct}</div>
-    <button class="startAgain" onclick="showPage(GamePage.StartPage)">PLAY AGAIN</button>
```

Comment:
thats why start button not appearing seems like you deleted it by mistake while merging